### ISSUE TITLE
Compatibility chart for README

### ISSUE THREAD

**Issue by computerlyrik** [mob41/broadlink-java-api#3]
Should we add a documentation, which devices have been tested successfully?
Ideally with the last working version and the user who tested it?

**Comment by a1aw** [comment]
You are right. We should write a clear documentation and tutorials that how to use the library.

What should we use for the documentation? Using the GitHub Wiki or GitHub Pages (with Bootstrap temptates? like https://mob41.github.io/ev3dev-lang-java)

**Comment by a1aw** [comment]
@computerlyrik What do you think?

**Comment by computerlyrik** [comment]
New standard for me. Seems nice. Just start with a template. 
Will be great if the doc source is kept within the original repository. (some Docu-Framworks don't)
